app/models/person_edition.rb

```diff
@@ -18,25 +18,26 @@ class PersonEdition < Edition
   field    :linkedin,         type: String
   field    :github,           type: String
   field    :node,             type: String
-  
+
   attaches_with_metadata :image
 
   GOVSPEAK_FIELDS = Edition::GOVSPEAK_FIELDS + [:description]
 
-  clone_fields :honorific_prefix, :honorific_suffix, :affiliation, :role, :description, :url,  
+  clone_fields :honorific_prefix, :honorific_suffix, :affiliation, :role, :description, :url,
                :telephone, :email, :twitter, :linkedin, :github, :node
-  
+
   def whole_body
     description
   end
 
   def rendering_app
     "www"
   end
-  
+
   def rendering_path
-    tag_to_rendering_path "team" => "team"
+    tag_to_rendering_path "team" => "team",
+                          "summit-speaker" => "summit/speakers"
```

Comment:
Should it be `summit/speakers` or should it be `summit/2016/speakers`? Obviously the 2016 would be a hardcoded hack, but probably one we won't have to deal with again once this is thrown away. Or, we have a special tag for 2016 speakers.